src/sys/resource.rs

```diff
@@ -167,7 +171,7 @@ libc_enum! {
         /// or used by all of this user id's processes.
         RLIMIT_SWAP,
 
-        #[cfg(target_os = "freebsd")]
+        #[cfg(any(target_os = "freebsd"))]
```

Comment:
You should revert this change.  It's not useful, and I think Clippy will complain about it.

Reply:
Sorry for this one, it was not my intention to push it. Its reverted.